docs/trex_sandbox_publish.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Publish Sandbox Extensions
+layout: docs
+---
+
+> *\*\* This is preliminary information and is subject to change \*\**
+
+Sandboxed Extensions are Tableau dashboard extensions that are not permitted to external network requests. Sandboxed Extensions are hosted by Tableau and run inside an environment that prevents communication with anything except the Tableau hosting server. Sandboxed Extensions are supported in Tableau 2019.4 and later. Available for testing with Tableau 2019.3.
+
+## Before publishing your extension
+
+Be sure to follow the guidelines and requirements to [Create and Test Sandboxed Extensions]({{site.baseurl}}/docs/trex_sandbox_test.html). Make sure your extension works as expected in the development environment. The development environment replicates the Tableau Sandboxed Extension Hosting Cloud Service. If your extension works in the development environment, it will work when you submit the extension package to Tableau for publication.
+
+## Submit your extension for publication
+
+To publish a Sandbox Extension, you need to contact Tableau and provide your extension package. This package includes all files and resources that your extension depends on. For more information, contact [Email link]() (*coming soon*)
```

Comment:
I spoke with @KeshiaRose and we'll be using the gallery submission form for publishing:
https://airtable.com/shr6WBBTEuZIpZ3Gj

Reply:
Updated gallery submission form link:
https://tabsoft.co/gallerysubmit

Reply:
Thanks. I've updated that section and use the tabsoft.co link.